.env.example

```
@@ -1,3 +1,9 @@
 GROQ_API_KEY=<add-groq-api-key-here>
 FIRECRAWL_API_KEY=<add-firecrawl-api-key-here>
-SERPER_DEV_API_KEY=<add-serper-dev-api-key-here>
\ No newline at end of file
+SERPER_DEV_API_KEY=<add-serper-dev-api-key-here>
+
+# Fine-tuned task-specific models (leave empty to use default LLM_MODEL)
+# Set these after running scripts/training/register_models.py
+QUERY_PARSER_MODEL=
+EXPLAINER_MODEL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPLAINER_MODEL key should go before the QUERY_PARSER_MODEL key

(UnorderedKey)